inst/rmarkdown/templates/vetiver_dashboard/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,91 @@
+---
+title: "Monitor model"
+output: 
+  vetiver::vetiver_dashboard:
+    board: !expr pins::board_rsconnect()
+    name: 'julia.silge/hotel_rf'
+    storyboard: true
+    source_code: 'https://vetiver.rstudio.com'
+    display_pins: true
+params:
+    board: !r pins::board_rsconnect()
+    name: 'julia.silge/hotel_rf'
+    version: NULL
```

Comment:
One of the big questions I have about this PR is how to avoid having all these things twice.

We need the `board` and `name` in the output format function because that is how the link for the pin is automatically generated, but we also need these in the `params` because they are used in the monitoring functions like `vetiver_pin_metrics()`.